### ISSUE TITLE
<FocusTrap /> fails to render when doing SSR

### ISSUE THREAD

**Issue by magicmark** [davidtheclark/focus-trap-react#31]
Hi @davidtheclark, thanks for the awesome project!

Something we noticed is that when using `<FocusTrap />` in an SSR environment, we get the following stack trace:

```
   ReferenceError: Element is not defined
     
     at Object.<anonymous> (node_modules/tabbable/index.js:14:15)
     at Object.<anonymous> (node_modules/focus-trap/index.js:1:105)
     at Object.<anonymous> (node_modules/y-modal/node_modules/focus-trap-react/dist/focus-trap-react.js:12:23)
```

This is because of this line https://github.com/davidtheclark/tabbable/blob/e8ecf52617fe97066361356070ae42364e089c8b/index.js#L14 - `Element` only exists on `window`, and not in a node environment

Clearly this component only really makes sense in a browser environment, and the answer here is the detect when we're in SSR and stub out the component as necessary.

I'm wondering if we could do this automagically behind the scenes tho, so this is transparent to application developers? Maybe we could just render a dumb `<div>` (or whatever element was specified) if we're in a SSR environment.

Would you be open to such a PR? Thanks!

**Comment by davidtheclark** [comment]
@magicmark This should be fixed by https://github.com/davidtheclark/tabbable/pull/33, which was released in tabbable 3.1.1. So in theory you could get the fix now. I'd be open to PRs updating the versions to enforce that, but might not get to it myself for a few days.

**Comment by magicmark** [comment]
Oh rad! Thank you!
